packages/cms-base/components/public/cms/element/CmsElementCategoryNavigation.vue

```diff
@@ -1,33 +1,65 @@
 <script setup lang="ts">
+import type { Ref } from "vue";
+import type { ClientApiError } from "@shopware-pwa/types";
 import SwCategoryNavigation from "../../../SwCategoryNavigation.vue";
 import deepMerge from "../../../../helpers/deepMerge";
 import getTranslations from "../../../../helpers/getTranslations";
+import { getTranslatedProperty } from "@shopware-pwa/helpers-next";
 import { useCategory, useNavigation } from "#imports";
-import { onMounted, computed } from "vue";
+import { onMounted, ref } from "vue";
 import type { Schemas } from "#shopware";
 
 type Translations = {
   listing: {
     category: string;
+    subCategory: string;
+    subCategoryOf: string;
+    loading: string;
     categories: string;
+    subCategories: string;
   };
 };
 
 let translations: Translations = {
   listing: {
     category: "Category",
+    subCategory: "Sub-category",
+    subCategoryOf: "of",
+    loading: "Loading ...",
     categories: "Categories",
+    subCategories: "Sub-categories",
   },
 };
 
 const globalTranslations = getTranslations();
 translations = deepMerge(translations, globalTranslations) as Translations;
 
 const { category: activeCategory } = useCategory();
-const { loadNavigationElements, navigationElements } = useNavigation();
-const navigations = computed(() => {
+const loading: Ref<boolean> = ref(true);
+const useSubCategories: Ref<boolean> = ref(true); // could be passed maybe as a prop in the future
```

Comment:
it doesn't not change, no need for ref. Also useX is implicating composable, my proposition
```suggestion
const flagAllowSubcategories = true;
```